tests/roxctl/bats-tests/local/deployment-bundles-psps.bats

```diff
@@ -32,32 +32,32 @@ central_bundle_psp_disabled() {
     shift
     local bundle_dir="${out_dir}/bundle-central-${RANDOM}-${RANDOM}-${RANDOM}"
     roxctl-release central generate "${cluster_type}" pvc "$@" --output-dir="${bundle_dir}"
-    run grep -rq "kind: PodSecurityPolicy" "${bundle_dir}"
+    run grep -rq "kind: PodSecurityPolicy" "${bundle_dir}/central"
     assert_failure
 }
 
 # Testing: central generate k8s
 @test "PodSecurityPolicies can be disabled for central deployment bundle (k8s)" {
-    central_bundle_psp_enabled k8s --enable-pod-security-policies=false
+    central_bundle_psp_disabled k8s --enable-pod-security-policies=false
```

Comment:
This was also a bug in the old testing code, which wasn't noticed because of the previous bugs. :-(